test/TestExpectations.json

```diff
@@ -1697,6 +1697,24 @@
     "parameters": ["cdp", "firefox"],
     "expectations": ["FAIL"]
   },
+  {
+    "testIdPattern": "[device-request-prompt.spec] device request prompt can be opened",
+    "platforms": ["darwin", "linux", "win32"],
+    "parameters": ["chrome", "headless", "cdp"],
```

Comment:
Please remove the `cdp` as that's the protocol part, and from what I understand it's not working for `--headless` .
We should disable this test in every instance of it.

Reply:
removed in 1dca498220ed7f2687ccba41b75ba7035c2f1751

Reply:
@Lightning00Blade let me know if there other changes i need to make to disable this properly for headless. should i also do something like this?
https://github.com/puppeteer/puppeteer/blob/20ee325e623a4576daeaae4ed3e3831d37782043/test/src/launcher.spec.ts#L571

Reply:
Not necessary, as `isHeadless` will return true for both `--headless=true` and [`--headless="new"`](https://developer.chrome.com/articles/new-headless/) and given the second is the same as Headful we should run the test there.